### PR TITLE
update README and cleanup

### PR DESCRIPTION
**Pull request by rhagarty**
- cleaned up some screen shots
- removed some unneeded files/dirs
- consolidated all data files into /data
- moved 'Deploy to IBM Cloud' steps to below setup of services
- modified DemoServlet.java to read env variable 'MODEL_ID'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,7 +311,34 @@ curl -u "$username":"$password" \
...
+2. In Toolchains, click on Delivery Pipeline to watch while the app is deployed. Once deployed, the app can be viewed by clicking 'View app'.
+
+![](doc/source/images/toolchain-pipeline.png)
+
+3. To see the app and services created and configured for this journey, use the IBM Cloud dashboard. The app is named `sms-analysis-with-wks` with a unique suffix. The following service is created:
+    * sms-nlu-service
+
+4. One last step is required to configure the app to run with your WKS model.
+
+In the IBM Cloud dashboard, find the app that was created. Click on `Runtime` on the menu and navigate to the `Environment variables` tab.
```
Comment (markstur) [inline README.md]: Probably mention that you click on the app to get into manage before you can find Runtime (?)

File: README.md [hunk 2]
```diff
@@ -336,7 +311,34 @@ curl -u "$username":"$password" \
...
+
+!!! Add screen shot
```
Comment (markstur) [inline README.md]: TODO: did you want to commit this?

File: README.md [hunk 3]
```diff
@@ -336,7 +311,34 @@ curl -u "$username":"$password" \
...
+
+Set the `MODEL_ID` to the value you saved in Step 9.
```
Comment (markstur) [inline README.md]: I think the code wanted modelId.  Need to check.

File: manifest.yml [hunk 4]
```diff
@@ -12,3 +12,5 @@ applications:
   buildpack: liberty-for-java
   services:
   - sms_nlu_service
+env:
+  MODEL_ID: placeholder
```
Comment (markstur) [inline manifest.yml:16]: I think this is modelId in the code (checking).

**Review by markstur — CHANGES_REQUESTED** [review]
See comments.  Checking on modelId and looking at README format still.

File: README.md (hunk 1 above)
Comment (rhagarty) [reply README.md]: can do

File: README.md (hunk 2 above)
Comment (rhagarty) [reply README.md]: placeholder - coming in next commit

File: README.md (hunk 3 above)
Comment (rhagarty) [reply README.md]: I'm will change the code to match the uppercase version. I still need to add the logic to read from a config file.

File: manifest.yml (hunk 4 above)
Comment (rhagarty) [reply manifest.yml:16]: see above

**Review by markstur — APPROVED** [review]
LGTM

**Pull request merged**